src/main/java/mate/academy/internetshop/controller/bucketcontroller/AddItemToBucketController.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.internetshop.controller.bucketcontroller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import mate.academy.internetshop.lib.Inject;
+import mate.academy.internetshop.model.Bucket;
+import mate.academy.internetshop.model.Item;
+import mate.academy.internetshop.service.BucketService;
+import mate.academy.internetshop.service.ItemService;
+import mate.academy.internetshop.service.UserService;
+
+public class AddItemToBucketController extends HttpServlet {
+    private static final long USER_ID = 1L;
```

Comment:
```suggestion
    private static final Long USER_ID = 1L;
```
краще використовувати такий тип скрізь

Reply:
+